core/src/main/scala/org/bitcoins/core/wallet/utxo/TxoState.scala

```diff
@@ -19,7 +19,10 @@ object TxoState extends StringFactory[TxoState] {
     */
   final case object ImmatureCoinbase extends TxoState
 
-  /** Means we have received funds to this utxo, but they are not confirmed */
+  /** Means we have received funds to this utxo, and they have not been confirmed in a block */
+  final case object BroadcastReceived extends ReceivedState
+
+  /** Means we have received funds to this utxo, and they have not reached our confirmation threshold */
```

Comment:
If I understand, we have received some confirmations, but not our confirmation threshold.

I just want to make a clear distinction of when the `BroadcastReceived` state ends and the `PendingConfirmationsReceived` state begins.